src/System.IO.Ports/src/Interop/Unix/Interop.Termios.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO.Ports;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Termios
+    {
+        internal enum Signals
+        {
+            SignalDtr = 1,
+            SignalDsr = 2,
+            SignalRts = 3,
+            SignalCts = 4,
+            SignalDcd = 5,
+        }
+
+        internal enum Queue
+        {
+            ReceiveQueue = 1,
+            SendQueue = 2,
+            AllQueues = 3,
+        };
+
+        [DllImport(Libraries.IOPortsNative, EntryPoint = "TermiosReset", SetLastError = true)]
```

Comment:
Looking at other mini-PALs we have, it looks like the native functions should be named with underscore between the `Termios` and the rest of the function name to match the convention used elsewhere. So e.g. `Termios_Reset`.